modules/nextflow/src/main/groovy/nextflow/database/DBSql.groovy

```diff
@@ -0,0 +1,87 @@
+package nextflow.database
+
+import groovy.util.logging.Slf4j
+import groovy.sql.Sql
+import groovyx.gpars.dataflow.DataflowQueue
+import groovyx.gpars.dataflow.DataflowWriteChannel
+import nextflow.Channel
+import nextflow.extension.CH
+
+@Slf4j
+class DBSql {
```

Comment:
It would be better to annotate this as `@CompileStatic` https://stackoverflow.com/a/29611007/395921

Reply:
Got it.